resources/zib/zib-MedicationDispense.xml

```diff
@@ -0,0 +1,222 @@
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A dispense is the delivery of an amount of medicine to the patient, their administrator or their representative." />
+  <purpose value="This MedicationDispense resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) MedicationDispense v2.0.2 (2020)](https://zibs.nl/wiki/MedicationDispense-v2.0.2(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="medicationdispense-v2.0.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/MedicationDispense-v2.0.2(2020EN)" />
+    <name value="zib MedicationDispense-v2.0.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="MedicationDispense" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/MedicationDispense" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="MedicationDispense">
+      <path value="MedicationDispense" />
+      <short value="MedicationDispense" />
+      <alias value="Medicatieverstrekking" />
+      <mapping>
+        <identity value="medicationdispense-v2.0.2-2020EN" />
+        <map value="NL-CM:9.9.20270" />
+        <comment value="MedicationDispense" />
+      </mapping>
+    </element>
+    <element id="MedicationDispense.extension">
+      <path value="MedicationDispense.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationDispense.extension:medicationDispenseAdditionalInformation">
+      <path value="MedicationDispense.extension" />
+      <sliceName value="medicationDispenseAdditionalInformation" />
+      <min value="0" />
+      <max value="*" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationDispense.AdditionalInformation" />
+      </type>
+    </element>
+    <element id="MedicationDispense.extension:requestDate">
+      <path value="MedicationDispense.extension" />
+      <sliceName value="requestDate" />
+      <min value="0" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationDispense.RequestDate" />
+      </type>
+    </element>
+    <element id="MedicationDispense.extension:category">

```

Comment:
This seems backward. `.category` has a _preferred_ ValueSet, there should be a very good reason to use something else there. I don't see that reason: SNOMED|373784005 is defined as "Dispensing medication (procedure)" -- that's basically what a MedicationResource represents.

Reply:
https://bits.nictiz.nl/browse/MP-327 
Let's discuss Monday.

Reply:
Moved it to the nl-core-profiles, added more documentation on it.